dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -56,7 +56,7 @@ if [[ ! -z "$NAMESPACE_ID" ]]; then
    update_default_gw 6
 fi
 
-if [ -z "$CONFIG_TYPE" ] || [ "$CONFIG_TYPE" == "separated" ]; then
+if  [ "$CONFIG_TYPE" == "separated" ]; then
```

Comment:
Should we completely remove "separated" mode? I believe with db_migrator, any old configuration will be migrated to unified.
Keeping separated would result in "non integrated" mode in vtysh which wouldn't be supported by FRR community going forward.